java/vector/src/main/codegen/data/ValueVectorTypes.tdd

```diff
@@ -116,7 +100,7 @@
         {
           class: "Decimal",
           maxPrecisionDigits: 38, nDecimalDigits: 4, friendlyType: "BigDecimal",
-          typeParams: [ {name: "scale", type: "int"}, { name: "precision", type: "int"}],
+          typeParams: [ { name: "precision", type: "int"}, {name: "scale", type: "int"} ],
```

Comment:
These were previously flipped in ARROW-1091 and then flipped again in ARROW-1092.  I thought they were right already, are they not?

Reply:
The reverse ordering the these typeParams are driving me nuts... I changed it such that they follow the same order everywhere. Also see: https://github.com/apache/arrow/pull/1330/files#r152436539

The end results are the same because flipped twice..